cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/schedule/trigger/ProgramStatusTriggerBuilder.java

```diff
@@ -57,11 +47,8 @@ public ProgramStatusTriggerBuilder(@Nullable String programNamespace, @Nullable
   @Override
   public ProgramStatusTrigger build(String namespace, String applicationName, String applicationVersion) {
```

Comment:
So just to confirm, a ProgramStatusTriggerBuilder is only used when we are inheriting all of the deployed application environment attributes? So the namespace, applicationName, and applicationVersion? Not any combination of them?